spec/features/poll_spec.rb

```diff
@@ -143,3 +143,28 @@
     expect(current_page).to have_status
   end
 end
+
+feature 'when cast a vote' do
+  scenario 'can acces vot by uniq url', js: true do
+    current_page = ShowPollPage.new
+    current_page.visit_page
+
+    expect(current_page).to have_expected_title
```

Comment:
Do you like this implicit way of checking the feature?
Because I personally — not so much.

It's really hard to figure out what's the expected title for the page, so it's actually harder to fix this in case of failure.
`expect(current_page.title).to eq "Some explicit string"` seems to hit the sweet spot for me — it encapsulate all the repetitive HTML parsing inside the `ShowPollPage` object, but still keeps all the variable values visible in spec itself.